softdevice_controller/Kconfig

```
@@ -31,8 +31,8 @@ config BT_CTLR_LLPM
 
 choice BT_LL_CHOICE
 	prompt "Bluetooth Link Layer Selection"
-	default BT_LL_SW_SPLIT if BT_MESH
-	default BT_LL_SOFTDEVICE
+	default BT_LL_SOFTDEVICE if BT_MESH
+	default BT_LL_SW_SPLIT
```

You would be changing the default for non-mesh. So if you are removing the mesh default to SPLIT it should look like this:
```suggestion
	default BT_LL_SOFTDEVICE
```